migrations/20211001142037-create-media.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('media', {
+      file: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        primaryKey: true,
+      },
+      url: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      org_id: {
+        type: Sequelize.STRING,
+        allowNull: false,
```

Comment:
Missing FK `references` object property

Reply:
This is not resolved. The created table has no FK.
<img width="831" alt="imagem" src="https://user-images.githubusercontent.com/36140182/135875869-caa4559f-d9f9-4e26-99bf-151bbb1fd098.png">